src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-form/cephfs-subvolume-form.component.html

```diff
@@ -43,6 +43,26 @@
           </div>
         </div>
 
+          <!--Subvolume Group name -->
+          <div *ngIf="!this.isEdit"
+               class="form-group row">
+            <label class="cd-col-form-label"
+                   for="subvolumeGroupName"
+                   i18n>Subvolume group name
+            </label>
+            <div class="cd-col-form-input">
+              <select class="form-select"
+                      id="subvolumeGroupName"
+                      name="subvolumeGroupName"
+                      formControlName="subvolumeGroupName">
+                <ng-container *ngIf="subVolumeGroups$ | async as subvolumeGroups">
```

Comment:
instead of a separate `ng-container` why not place this at the `form-control` div and we could remove this `ng-container`.